djapm/apm/contrib.py

```diff
@@ -19,10 +19,10 @@ def _contribute_to_request(
         data = rest_request.data
         view_name = get_view_name(view)
         if view_name == "Function":
-            view_name = ".".join([view.__module__, view.__name__])
+            view_name = ".".join([view.__module__, view.__class__.__name__])
```

Comment:
I think that i prefer this:
`view_name = ".".join([view.__module__, getattr(view, "__name__", view.__class__.__name__]))`

Reply:
I updated the branch with this code and it works. So it's good for me.